.github/workflows/merge_queue.yml

```diff
@@ -60,11 +60,6 @@ jobs:
       run_command: |
           python3 style_check.py
```

Comment:
this misses [--no-push](https://github.com/ClickHouse/ClickHouse/blob/5445f55e62d196498359aeff3e7a8de17d61d84d/tests/ci/style_check.py#L71) when the key is removed